builders/super-builder.js

```diff
@@ -33,7 +33,7 @@ module.exports = {
     if (baseContract !== undefined) {
       if(!baseContract.ignore) {
         // If file is not ignored, then add link to file as well
-        builder.push(`[${baseContract.contractName}.${node.name}](${baseContract.contractName}.md#${node.name.toLowerCase()})`);
+        builder.push((!node.name)?`[${baseContract.contractName}.${node.name}](${baseContract.contractName}.md)`:`[${baseContract.contractName}.${node.name}](${baseContract.contractName}.md#${node.name.toLowerCase()})`);
```

Comment:
If we don't have node.name shouldn't we do this?
```suggestion
        builder.push((!node.name)?`[${baseContract.contractName}](${baseContract.contractName}.md)`:`[${baseContract.contractName}.${node.name}](${baseContract.contractName}.md#${node.name.toLowerCase()})`);
```

Reply:
Yes, we should. Good catch!! 👍🏼